src/api.cu

```diff
@@ -165,130 +168,163 @@ TRITONBACKEND_ModelInstanceExecute(
     TRITONBACKEND_ModelInstance* instance, TRITONBACKEND_Request** raw_requests,
     const uint32_t request_count)
 {
-  uint64_t all_start_time =
-      std::chrono::steady_clock::now().time_since_epoch().count();
-  std::size_t total_inference_count = 0;
-  std::vector<TRITONBACKEND_Response*> responses;
+  auto all_start_time =
+      uint64_t(std::chrono::steady_clock::now().time_since_epoch().count());
+
+  auto requests = std::vector<TRITONBACKEND_Request*>(
+      raw_requests, raw_requests + request_count);
+
+  auto instance_state = get_instance_state<ModelInstanceState>(*instance);
```

Comment:
Could you rephrase this as `auto *instance_state` so it's clear it's a pointer type?